src/services/user.methods.ts

```diff
@@ -0,0 +1,100 @@
+import { Router } from 'express';
+import { Op } from 'sequelize';
+import { userRouter } from '../controllers/user.routes';
+import { IBaseUser, IUser, UserModelAttributes } from "../interfaces/user.types";
+import { UserErrors } from '../models/user.model';
+import { initializeUserTable } from '../services/user.initialize';
+
+const users: IUser[] = [];
+
+const throwUserError = (type: UserErrors, param?: unknown): void => {
+    switch (type) {
+        case UserErrors.UserIdDoesNotExist:
+            throw new Error(`user with id ${param} doesn't exist`);
+        case UserErrors.UserRemoved:
+            throw new Error(`user with id ${param} exists but removed`);
+        case UserErrors.UserLoginDoesNotExist:
+            throw new Error(`user with login ${param} doesn't exist`)
+    }
```

Comment:
`switch` is a very complicated construction, it's better to avoid using it. It can be easily replaced with a object.

Reply:
not agreed, switch/case and enum are best friends